apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/BrevDataMapperRedigerbartUtfallVedtak.kt

```diff
@@ -164,14 +170,21 @@ class BrevDataMapperRedigerbartUtfallVedtak(
                             sakType,
                             vedtakType,
                         )
-                    VedtakType.ENDRING ->
-                        omstillingsstoenadEndring(
-                            brukerTokenInfo,
-                            behandlingId,
-                            virkningstidspunkt!!,
-                            sakType,
-                            vedtakType,
-                        )
+
+                    VedtakType.ENDRING -> {
+                        if (revurderingaarsak != null && revurderingaarsak == Revurderingaarsak.AARLIG_INNTEKTSJUSTERING) {
```

Comment:
Skal være nok med kun `revurderingaarsak == Revurderingaarsak.AARLIG_INNTEKTSJUSTERING`

Reply:
revurderingaarsak er faktisk nullable her. Jeg tror det er fordi det brukes for førstegasngsbehandling også og da finnes det ikke revurderingaarsak